src/hooks/context/history-context.js

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext, useReducer } from "react";
+import { historyReducer } from "../reducer/history-reducer";
```

Comment:
Just a suggestion: You can export all reducers from a single index.js file inside reducer folder. Can be done later as well while code refactoring

Reply:
Okay, I will do that.